pkgs/stdenv/darwin/test-bootstrap-tools.nix

```diff
@@ -30,7 +31,7 @@ builtins.derivation {
     mkdir -p $out/bin
 
     for exe in $tools/bin/*; do
-      [[ $exe =~ bunzip2|codesign.*|false|install_name_tool|ld|lipo|pbzx|ranlib|rewrite-tbd|sigtool ]] && continue
+      [[ $exe =~ bunzip2|codesign.*|false|install_name_tool|jq|ld|lipo|pbzx|ranlib|sigtool|tapi ]] && continue
```

Comment:
jq should support `jq --version` and return 0 so the test passes. is there a way to run `tapi` that will return 0, maybe `tapi -h` or `tapi -v` ? if so can just add below `sigtool -h` so all programs get run in the test.

Reply:
`tapi` supports `--version`

```shell
./tapi --version
TAPI version 15.0.0 (tapi-1500.0.12.3)
```

Reply:
Oh, right. Exclusion list. I’ll drop jq and tapi from it.